Packs/MicrosoftECM/pack_metadata.json

```diff
@@ -13,4 +13,4 @@
     "tags": [],
     "useCases": [],
     "keywords": []
```

Comment:
@hod-alpert Let's add to the keywords: "ecm", "ms ecm", "sccm", "ms sccm" so it will come up in searches better.